src/App.js

```diff
@@ -1,15 +1,41 @@
 import React, { Component } from "react";
+import moment from 'moment'
 import {
   ReactiveBase,
-  DataSearch,
+  SearchBox,
   NumberBox,
   DateRange,
-  RangeSlider
+  RangeInput,
+  SelectedFilters
 } from "@appbaseio/reactivesearch";
+
+import './App.css'
 import { ReactiveGoogleMap } from "@appbaseio/reactivemaps";
-import "./App.css";
 
 class App extends Component {
+
+	dateQuery(value) {
```

Comment:
@SavvyShah Can you try not using this customQuery? I doubt it is required now.

Reply:
No it doesn't work. I tried using it without customQuery.

Reply:
Tried different values for `queryFormat`